com.github.Bavarder.Bavarder.json

```diff
@@ -0,0 +1,54 @@
+{
+    "app-id" : "com.github.Bavarder.Bavarder",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "bavarder",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        "pypi-dependencies.json",
+        {
+            "name" : "blueprint-compiler",
```

Comment:
this should have a cleanup remove to remove it all since it's only needed at build time.